bash_completion.sh

```diff
@@ -147,7 +147,7 @@ _buildtest ()
       ;;
 
     report|rt)
-      local opts="--filter --format --help --helpfilter --helpformat --latest --no-header --oldest --pager --terse  -h -n -t clear list summary"
+      local opts="--filter --format --failure --help --helpfilter --helpformat --latest --no-header --oldest --pager --terse  -h -n -t clear list summary"
```

Comment:
Just a nitpick, but i have this alphabetized with long options, short options and subcommands. Can you put `--failure` in front of `--filter`